### ISSUE TITLE
SoftmaxOutput - how to get probabilities?

### ISSUE THREAD

**Issue by smissan** [apache/incubator-mxnet#6179]
It looks like softmax output by default returns 1 for the class with highest probability, and 0 for the rest. Is there a way to get "true" probability to understand the confidence of classification? We followed the LeNet example in CPP wrapper.

**Comment by jiajiechen** [comment]
Can you provide a code snippet and std output? If you are talking about the API `SoftmaxOutput`, the forward pass should give you the probabilities.

**Comment by yajiedesign** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to reopen if this is still an active issue. Thanks!

**Comment by beeva-enriqueotero** [comment]
In order to apply typical crossentropy function as eval_metric you need to access to these probabilities. Otherwise training loss and validation loss will represent different metrics when using SoftmaxOutput !
